hadoop-common-project/hadoop-common/src/site/markdown/filesystem/filesystem.md

```diff
@@ -88,7 +88,7 @@ Get the status of a path
             stat.length = len(FS.Files[p])
             stat.isdir = False
             stat.blockSize > 0
-        elif isDir(FS, p) :
+        elif isDirectory(FS, p) :
```

Comment:
`isDir(FS, Path)` is the function in the model; 'isDirectory()` path the published method in Filesystem.

Reply:
Indeed, missed that. Also changed line 67 to reflect that.